build.gradle.kts

```diff
@@ -23,7 +23,7 @@ plugins {
   id("java")
   // Dependencies are locked at this version to work with JDK 11 on CI.
   id("org.jetbrains.kotlin.jvm") version "1.9.10"
-  id("org.jetbrains.intellij") version "1.15.0"
+  id("org.jetbrains.intellij") version "1.17.0"
```

Comment:
I cannot see anything important for us.

https://github.com/JetBrains/gradle-intellij-plugin/releases